### PR TITLE
Adding two new functions and corresponding tests

### PR DESCRIPTION
**Pull request by SandeepThokala**
new functions in `sequence_info.py` for handling [detectOVRF #10](https://github.com/PoonLab/detectOVRF/issues/10)

### CODE REVIEW

File: tests/test_sequence_info.py [hunk 1]
```diff
@@ -4792,6 +4851,28 @@ def testGetRightLeftNT(self):
...
         self.assertEqual(result.state, seq5[len(seq5) - 1])
 
+    def testGetOverlappingInfo(self):
+        expected = \
+            {'0_12': {'[[0, 3]]': {'overlaps_with': [],
+                                   'len': 4,
+                                   'codons': [[0, 1]]},
+                      '[[4, 11]]': {'overlaps_with': ['4_16'],
+                                    'len': 8,
+                                    'codons': [[2, 3]]}},
```
Comment (horaciobam) [inline tests/test_sequence_info.py:4861]: The codons on region `[[4, 11]]` should be `[[2,4]]` instead of `[[2,3]]

**Review by horaciobam — COMMENTED** [review]
I think that we need to clarify how the `codon` lists are storing the information. Is it inclusive or not?

For example, the region `[[4, 11]]` of orf `0_12` includes the nucleotides `4,5,6; 7,8,9; 10,11`. Therefore, the `codons` coordinates should be `[[2, 4]]` , meaning that it has the second and third codon of the `orf`, but not the fourth. Since the region is not multiple of three, the last nucleotide required to get the fourth codon is missing.

**Comment by SandeepThokala** [comment]
According to the orfs information provided in the `setUp` function of `TestSequence5`
The region `[4, 12]` looks like..`4,5; 6,7,8; 9,10,11; 12` in terms of codons

and now the codon list is specifying the first and last indices of complete codons (non partial) in that region both `inclusive`
So we're getting `[2, 3]` as a result.

**Comment by ArtPoon** [comment]
@horaciobam can you please weigh in on the above?

**Pull request merged**